app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,492 @@
+  <string name="app_name">Phoenix (Testnet)</string>
+
+  <!-- //////////////// system notification channels //////////////// -->
+
+  <string name="notification_channels_watcher_title">Verifica on-chain dei canali</string>
+  <string name="notification_channels_watcher_desc">Notifica se devi aprire l\'app. Dovrebbe avere alta priorità.</string>
+
+  <!-- //////////////// init wallet / automated wallet creation //////////////// -->
+
+  <string name="autocreate_generating">Creazione del wallet…</string>
+  <string name="autocreate_error">Non è stato possibile creare il tuo wallet.\nMotivo: %1$s</string>
+
+  <!-- //////////////// init wallet / automated wallet creation //////////////// -->
+
+  <string name="restore_disclaimer_message">Non importare un seed che non è stato creato da quest\'app.\n\nInoltre, assicurati di non avere un altra app Phoenix aperta con lo stesso seed.</string>
+  <string name="restore_disclaimer_checkbox">Ho capito.</string>
+  <string name="restore_disclaimer_next">Avanti</string>
+  <string name="restore_instructions">Il seed del tuo wallet è una lista di 12 parole inglesi. Nota bene che l\'uso di una passphrase non è supportato.</string>
+  <string name="restore_mnemonics_label">Usa la seguente tastiera per inserire il tuo seed:</string>
+  <string name="restore_in_progress">Ripristino del wallet…</string>
+  <string name="restore_import_button">Importa</string>
+  <string name="restore_error">Seed invalido.\n\nPer favore prova di nuovo</string>
+
+  <!-- //////////////// init wallet / create seed //////////////// -->
+
+  <string name="newseed_instructions">Questa lista di parole è la chiave del tuo wallet. Salvala in un luogo sicuro (non su questo telefono).</string>
+  <string name="newseed_words_td"><![CDATA[<td><font color="rgba(100,100,100,.3)"><small>#%1$d</small></font> <b>%2$s</b></td>]]></string>
+  <string name="newseed_generating_seed">Generazione del seed in corso</string>
+  <string name="newseed_seed_saved_checkbox">Ho fatto il backup di questa lista di parole.</string>
+  <string name="newseed_words_sole_recourse_checkbox">Ho capito che questa lista è la chiave dei miei fondi.\nSono l\'unico possessore.</string>
+  <string name="newseed_next">Avanti</string>
+
+  <!-- //////////////// Pin dialog //////////////////// -->
+
+  <string name="pindialog_title_default">Per favore inserisci il PIN</string>
+  <string name="pindialog_pwd_placeholder">●●●●●●</string>
+  <string name="pindialog_error">Il codice pin deve avere 6 cifre.</string>
+  <string name="pindialog_error_donotmatch">PIN non corretto.</string>
+
+  <!-- //////////////// Biometric dialog //////////////////// -->
+
+  <string name="biometricprompt_title">Wallet bloccato</string>
+  <string name="biometricprompt_negative">Sblocca con il PIN</string>
+
+  <!-- //////////////// placeholders ////////////////-->
+
+  <string name="lipsum_short">Etiam porttitor egestas faucibus. Curabitur condimentum eros non ipsum elementum egestas.</string>
```

Comment:
There are some texts like these placeholders that should not be translated.

The should be removed from the translated file, and in the original default `strings.xml`, these fields must have the `translatable="false"` tag.